### ISSUE TITLE
Italian translation

### ISSUE THREAD

**Issue by Viktormax** [Yummypets/YPImagePicker#111]
Wonderful job!

[YPImagePicker-master plus ita version.zip](https://github.com/Yummypets/YPImagePicker/files/2000788/YPImagePicker-master.plus.ita.version.zip)

**Comment by s4cha** [comment]
@Viktormax Thank YOU so much, we will add this to the next release :) Actually I'll come back to you to check the translation for the newly introduced wordings.

**Comment by Viktormax** [comment]
OK, when u’ll have some news, I’ll be glad to help with Italian translation.

V.

**Comment by s4cha** [comment]
Well if you could check the following wordings that would be 👌  !
Some also need tranlsation (marked with 🔴 )
```
"YPImagePickerAlbums" = "Album";
"YPImagePickerCancel" = "Annula";
"YPImagePickerCover" = "Copertina";
"YPImagePickerCrop" = "Ridimensiona";
"YPImagePickerDone" = "Fine";
"YPImagePickerFilter" = "Filtro";
"YPImagePickerLibrary" = "Libreria";
"YPImagePickerNext" = "Avanti";
"YPImagePickerOk" = "Ok";  🔴
"YPImagePickerPermissionDeniedPopupCancel" = "Annulla";
"YPImagePickerPermissionDeniedPopupGrantPermission" = "Autorizzazione fornita";
"YPImagePickerPermissionDeniedPopupMessage" = "E' necessario autorizzare l'accesso";
"YPImagePickerPermissionDeniedPopupTitle" = "Autorizzazione negata";
"YPImagePickerPhoto" = "Foto";
"YPImagePickerProcessing" = "Processing..";  🔴
"YPImagePickerSave" = "Save";  🔴
"YPImagePickerTrim" = "Taglia";
"YPImagePickerVideo" = "Video"; //ignore-same-translation-warning
"YPImagePickerVideoDurationTitle" = "Durata del video";
"YPImagePickerVideoTooLong" = "Carica un video inferiore a %@ secondi.";
"YPImagePickerVideoTooShort" = "Il video deve durare almeno %@ secondi";
"YPImagePickerWarningItemsLimit" = "Il limite è di %d foto o video";

```
Here are the English ones for comparison
```
"YPImagePickerAlbums" = "Albums";
"YPImagePickerCancel" = "Cancel";
"YPImagePickerCover" = "Cover";
"YPImagePickerCrop" = "Crop";
"YPImagePickerDone" = "Done";
"YPImagePickerFilter" = "Filter";
"YPImagePickerLibrary" = "Library";
"YPImagePickerNext" = "Next";
"YPImagePickerOk" = "Ok";
"YPImagePickerPermissionDeniedPopupCancel" = "Cancel";
"YPImagePickerPermissionDeniedPopupGrantPermission" = "Grant Permission";
"YPImagePickerPermissionDeniedPopupMessage" = "Please allow access";
"YPImagePickerPermissionDeniedPopupTitle" = "Permission denied";
"YPImagePickerPhoto" = "Photo";
"YPImagePickerProcessing" = "Processing..";
"YPImagePickerSave" = "Save";
"YPImagePickerTrim" = "Trim";
"YPImagePickerVideo" = "Video";
"YPImagePickerVideoDurationTitle" = "Video duration";
"YPImagePickerVideoTooLong" = "Pick a video less than %@ seconds long";
"YPImagePickerVideoTooShort" = "The video must be at least %@ seconds";
"YPImagePickerWarningItemsLimit" = "The limit is %d photos or videos";
```

Cheers :)

**Comment by s4cha** [comment]
The Italian translation is available in the latest release (3.0.0) :)
